tests/event/test_event_storage.py

```diff
@@ -0,0 +1,42 @@
+import random
+import string
+import tempfile
+import unittest
+from typing import Optional
+
+import pytest
+
+from hathor.event.storage.event_storage import EventStorage
+from hathor.event.storage.memory_storage import EventMemoryStorage
+from hathor.event.storage.rocksdb_storage import EventRocksDBStorage
+from hathor.storage.rocksdb_storage import RocksDBStorage
+from tests.utils import HAS_ROCKSDB, generate_mocked_event
+
+
+class EventStorageBaseTest():
```

Comment:
We usually have a different pattern for these, which would basically be:

```python
class EventStorageBaseTest(unittest.TestCase):
    __test__ = False

    # ...

class EventStorageMemoryTest(EventStorageBaseTest):
    __test__ = True
```

I'm not sure if there are advantages or disadvantages of doing it differently, but I'd prefer to stick to the current pattern for consistency.

Reply:
Done.